bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -73,26 +75,25 @@ public class JdbcBaseDAO {
     public final Map<String, String> sqlTypes = new HashMap<>();
 
     // Get Database Meta data
-    protected DbMetaData dbMeta;
-
-    protected String sqlPingDB;
-    protected String sqlGetDB;
-    protected String sqlIfTableExists;
-    protected String sqlCreateNewEntryInItemsTable;
-    protected String sqlCreateItemsTableIfNot;
-    protected String sqlDeleteItemsEntry;
-    protected String sqlGetItemIDTableNames;
-    protected String sqlGetItemTables;
-    protected String sqlCreateItemTable;
-    protected String sqlInsertItemValue;
+    protected @Nullable DbMetaData dbMeta;
+
+    protected String sqlPingDB = "SELECT 1";
```

Comment:
Shouldn't these be changed to `static final String`, and uppercased?

`protected static final String SQL_PING_DB = "SELECT 1";`

Reply:
They are changed by some of the subclasses, so they can't be final.

Reply:
OK